### PR TITLE
docs(api): update commands

### PR DESCRIPTION
**Pull request by snitin315**
update commands

Refers - https://github.com/webpack/webpack-cli/pull/2303 https://github.com/webpack/webpack-cli/pull/2357

### CODE REVIEW

File: src/content/api/cli.md [hunk 1]
```diff
@@ -38,17 +38,18 @@ Read the [installation guide](/guides/installation) if you don't already have we
 
 webpack-cli offers a variety of commands to make working with webpack easy. By default webpack ships with
 
-| Command   | Alias | Description                                                                                 |
-| --------- | ----- | ------------------------------------------------------------------------------------------- |
-| `init`    | c     | Initialize a new webpack configuration                                                      |
-| `migrate` | m     | Migrate a configuration to a new version                                                    |
```
Comment (alexander-akait) [inline src/content/api/cli.md:44]: Goo catch, migrate is deprecated, so let's remove it from docs :+1:

File: src/content/api/cli.md [hunk 2]
```diff
@@ -38,17 +38,17 @@ Read the [installation guide](/guides/installation) if you don't already have we
...
+| ------------ | -------------- | ----------------------------------------------------------------------- |
+| `build`      | `bundle`, `b`  | Run webpack (default command, can be omitted).                          |
+| `watch`      | `--watch`, `w` | Run webpack and watch for files changes.                                |
+| `loader`     | `l`            | Scaffold a loader repository                                            |
+| `plugin`     | `p`            | Scaffold a plugin repository                                            |
+| `info`       | `i`            | Output information about your system and dependencies                   |
+| `serve`      | `s`            | Run the webpack Dev Server                                              |
+| `configtest` | `t`            | Tests webpack configuration against validation errors                   |
+| `help`       | `h`            | Display help for commands and options                                   |
+| `version`    | `v`            | Output the version of `webpack`, `webpack-cli` and `webpack-dev-server` |
```
Comment (alexander-akait) [inline src/content/api/cli.md:51]: Should we add here `-v` and `-h` for help, also I think will be great to write syntax here, i.e. `build|bundle|b [options]`/`configtest|t [config-path]` and etc

File: src/content/api/cli.md [hunk 3]
```diff
@@ -38,17 +38,17 @@ Read the [installation guide](/guides/installation) if you don't already have we
...
-| `serve`   | s     | Run the webpack Dev Server                                                                  |
-| `bundle`  | b     | Run webpack (default command, can be omitted)                                               |
-| `help`    | h     | Display help for commands and options                                                       |
-| `version` | v     | Output the version of `webpack`, `webpack-cli` and `webpack-dev-server` |
+| Command      | Alias          | Description                                                             |
+| ------------ | -------------- | ----------------------------------------------------------------------- |
+| `build`      | `bundle`, `b`  | Run webpack (default command, can be omitted).                          |
+| `watch`      | `--watch`, `w` | Run webpack and watch for files changes.                                |
+| `loader`     | `l`            | Scaffold a loader repository                                            |
+| `plugin`     | `p`            | Scaffold a plugin repository                                            |
```
Comment (chenxsan) [inline src/content/api/cli.md:46]: When I ran these two commands,

image

Does it mean these two are not shipped by default as the `By default webpack ships with` claimed?

File: src/content/api/cli.md [hunk 4]
```diff
@@ -38,17 +38,17 @@ Read the [installation guide](/guides/installation) if you don't already have we
...
+| `info`       | `i`            | Output information about your system and dependencies                   |
+| `serve`      | `s`            | Run the webpack Dev Server                                              |
+| `configtest` | `t`            | Tests webpack configuration against validation errors                   |
```
Comment (chenxsan) [inline src/content/api/cli.md:49]:
```suggestion
| `configtest` | `t`            | Validate a webpack configuration                   |
```

File: src/content/api/cli.md (hunk 3 above)
Comment (alexander-akait) [reply src/content/api/cli.md:46]: Yes, `init` too, due to poor design, we are working on it

File: src/content/api/cli.md (hunk 4 above)
Comment (snitin315) [reply src/content/api/cli.md:49]: /cc @alexander-akait I think we should update this in webpack-cli too.

File: src/content/api/cli.md [hunk 5]
```diff
@@ -37,18 +37,19 @@ Read the [installation guide](/guides/installation) if you don't already have we
...
+| `configtest` | configtest|t [config-path]                | Validate a webpack configuration.                                               |
+| `help`       | help|h [command] [option]                 | Display help for commands and options.                                          |
+| `info`       | info|i [options]                          | Outputs information about your system.                                          |
+| `init`       | init|c [scaffold...] [options]            | Initialize a new webpack configuration.                                         |
+| `loader`     | loader|l [output-path]                    | Scaffold a loader.                                                              |
+| `plugin`     | plugin|p [output-path]                    | Scaffold a plugin.                                                              |
+| `serve`      | serve|s [options]                         | Run the `webpack-dev-server`.                                                   |
+| `version`    | version|v [commands...]                   | Output the version number of `webpack`, `webpack-cli` and `webpack-dev-server`. |
```
Comment (chenxsan) [inline src/content/api/cli.md:51]:
```suggestion
| `version`    | version\|v [commands...]                   | Output the version number of `webpack`, `webpack-cli` and `webpack-dev-server`. |
```
You are using `|` inside a table cell and it needs to be escaped with a `\` as per https://github.github.com/gfm/#tables-extension-

Comment (snitin315) [reply src/content/api/cli.md]: Ah yes 👍🏻

**Review by chenxsan — APPROVED** [review]

**Pull request merged**